sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/credentials/ConfigurationClientCredentials.java

```diff
@@ -72,41 +76,49 @@ public URL baseUri() {
 
     /**
      * Gets a list of headers to add to a request to authenticate it to the Azure APp Configuration service.
+     *
      * @param url the request url
      * @param httpMethod the request HTTP method
      * @param contents the body content of the request
+     *
      * @return a flux of headers to add for authorization
+     *
      * @throws NoSuchAlgorithmException If the SHA-256 algorithm doesn't exist.
      */
-    public Mono<Map<String, String>> getAuthorizationHeadersAsync(URL url, String httpMethod, Flux<ByteBuffer> contents) {
-        return contents
-            .collect(() -> {
+    public Mono<Map<String, String>> getAuthorizationHeadersAsync(URL url, String httpMethod,
+                                                                  Flux<ByteBuffer> contents) {
+        return contents.collect(
+            () -> {
                 try {
                     return MessageDigest.getInstance("SHA-256");
                 } catch (NoSuchAlgorithmException e) {
                     throw logger.logExceptionAsError(Exceptions.propagate(e));
                 }
-            }, (messageDigest, byteBuffer) -> {
-                    if (messageDigest != null) {
-                        messageDigest.update(byteBuffer);
-                    }
-                })
-            .flatMap(messageDigest -> Mono.just(headerProvider.getAuthenticationHeaders(url, httpMethod, messageDigest)));
+            },
```

Comment:
Does checkstyle complain about this? I feel like I remember checkstyle wanting `(message, byteBuffer)` on the previous line.